thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2581,12 +2581,12 @@ void CMasterGraph::getFinalProgress()
             if (nodeDiskUsage > maxNodeDiskUsage)
             {
                 maxNodeDiskUsage = nodeDiskUsage;
-                maxNode = n;
+                maxNode = n+1;
```

Comment:
Is this number correct for multi channel.  Should we change our convention?  Should it be a maxChannel rather than a maxNode?